docs/relnotes.md

```diff
@@ -0,0 +1,122 @@
+---
+hide:
+  - navigation
+---
+
+# Release Notes
+
+This is the list of changes to Taipy releases as they were published.
+
+## Community edition: 1.1
+
+Published on 2022-06.
+
+[`taipy` 1.1](https://pypi.org/project/taipy/1.1.0/) contains the latest
+[`taipy-gui` 1.1](https://pypi.org/project/taipy-gui/1.1.0/), 
+[`taipy-core` 1.1](https://pypi.org/project/taipy-core/1.1.0/) and
+[`taipy-rest` 1.1](https://pypi.org/project/taipy-rest/1.1.0/) packages.
+
+
+#### Improvements
+
+**`taipy-gui`**<br/>1.1.0
+
+   - The `State^` instance can be initialized in a user-defined function. See the _on_init_
+     attribute of the `Gui^` class for more details.
+   - Page definitions and the bound variables can be isolated in a module to clarify the
+     application code organization.<br/>
+     See this [section](manuals/gui/binding.md#scope-of-variable-binding) for details.
+   - The [`chart`](manuals/gui/viselements/chart.md) control can display georeferenced data on top
+     of maps.<br/>
+     See this [example](manuals/gui/viselements/charts/others.md#plotting-on-a-map) for details.
+   - The [`input`](manuals/gui/viselements/input.md) and [`slider`](manuals/gui/viselements/slider.md)
+     controls have a new _change_delay_ property that lets you tune how fast you want to propagate
+     changes.<br/>
+     This allows for a better user experience.
+   - The [`input`](manuals/gui/viselements/input.md) control has a new _password_ property that, if True,
+     obscures the user input.
+   - The [`input`](manuals/gui/viselements/input.md), [`number`](manuals/gui/viselements/number.md) and
+     [`selector`](manuals/gui/viselements/selector.md) controls have a new _label_ property that lets you
+    display a label next to the control.
+   - The [`layout`](manuals/gui/viselements/layout.md) block has new syntax that makes it easier to define
+     a repetition of column definition.
+
+**`taipy-core`**<br/>1.1.0
+
+   - Execution modes: "_development_" mode (default) runs tasks in a synchronous way one task at
+     a time, while "_standalone_" mode runs tasks in an asynchronous and parallel way using
+     sub-processes.
+   - _Retry policy_ to read entities: the global configuration attribute _retry_read_entity_ indicates
+     the number of times Taipy will retry in case of error.
+   - Performance improvements when reading and writing entities.
+
+#### Significant bug fixes
+
+**`taipy-gui`**<br/>1.1.0
+
+   - Concurrency issues were fixed.
+   - Taipy supports HTTPS via reverse proxies.<br/>
+     See [issue](https://github.com/Avaiga/taipy-gui/issues/263).
+   - Modifying a variable multiple times in _on_change()_  is now supported.<br/>
```

Comment:
this bug was never public 
In 1.0, we could have an infinite loop by modifying a variable in the on_change function for that variable
In 1.1, we prevented the possible infinite loop

Reply:
Moved to Improvements.